aries-core/src/js/components/core/Identifier/Identifier.js

```diff
@@ -10,23 +9,25 @@ export const Identifier = ({
   subTitle,
   size,
   ...rest
-}) => (
-  <Box align="center" {...rest}>
-    {children}
-    <Box>
-      {level ? (
-        <Heading level={level} margin="none">
-          {title}
-        </Heading>
-      ) : (
-        <Text size={size} weight="bold" color="text-strong">
-          {title}
-        </Text>
-      )}
-      <Text size={size}>{subTitle}</Text>
+}) => {
+  return (
```

Comment:
```suggestion
}) => (
```
Can we change this to simplify the format?